packages/server/bootstrap.js

```diff
@@ -49,6 +45,12 @@ if (isTestEnv()) {
 
 dotenv.config({ path: `${packageRoot}/.env` })
 
+//knex is a singleton controlled by module so can't wait til app init
+const { initOpenTelemetry } = require('./otel')
```

Comment:
i'm assuming there wasnt a specific reason to put this stuff before dotenv init

Reply:
There is/was: otel didn't like other things being initialized before it.

Reply:
@adamhathcock even dotenv? sounds very odd, what exactly did it not like about it? note that even in its existing form there is stuff happening before it

Reply:
maybe not that but I just put it above most/everything to be sure